tools/credentials_test.go

```diff
@@ -27,22 +29,35 @@ var _ = Describe("Credentials", func() {
 				InBar: "in-bar",
 			},
 		}
-	})
 
-	JustBeforeEach(func() {
-		var err error
 		secretData, err = tools.ToCredentialsSecretData(credsObject)
```

Comment:
Invoking the method under test should be performed within the describe for that function. It should not be invoked in the global describe